src/components/Login.js

```diff
@@ -43,7 +43,9 @@ class Login extends React.Component {
             <div className="col-md-6 offset-md-3 col-xs-12">
               <h1 className="text-xs-center">Sign In</h1>
               <p className="text-xs-center">
-                <a>Need an account?</a>
+                <Link to="register">
+                  <a>Need an account?</a>
```

Comment:
If nested in `Link`, is the `<a>` still needed?

Reply:
maybe could do without it. Will check